app/controllers/admin/reports_controller.rb

```diff
@@ -6,6 +6,22 @@ class ReportsController < BaseController
 
     def index
       authorize :report, :index?
+
+      # We previously only supported searching by target account domain for
+      # reports, we now have more search options, but it's important that we
+      # don't break any saved queries people may have:
+      return redirect_to_new_filter if outdated_filter?
+
+      # If there isn't a status filter parameter, redirect to include the status parameter as unresolved,
+      # this ensures the "status" option menu always shows a highlighted option.
+      if filter_params.exclude? :status
+        if params.slice(*ReportFilter::DIRECT_KEYS).present?
+          return redirect_to admin_reports_path(filter_params.merge({ status: 'all' }))
```

Comment:
Previously if you went to a specific Account then either of "Reports by others" or "Made reports", the results shown would be "unresolved" reports, not all the reports like you'd expect.

<img width="558" alt="Screenshot 2024-08-16 at 23 32 38" src="https://github.com/user-attachments/assets/7c9eaf0b-5482-4ffb-be21-812c380a4d48">

Reply:
However, for `target_origin` without status, you probably want "unresolved" reports.